cesql/cesql_tck/negate_operator.yaml

```diff
@@ -14,7 +14,11 @@ tests:
     expression: --'10'
     result: 10
 
-  - name: Invalid boolean cast
+  - name: Minus with boolean cast
     expression: -TRUE
+    result: -1
+
+  - name: Minus with missing attribute
+    expression: -missing
     result: 0
-    error: cast
+    error: match
```

Comment:
What does "match' mean here - I don't see it listed as a valid error? I expected "missingAttribute"

Reply:
Tbh I'm not sure how I ended up with `match` here...

You are totally correct, it should be `missingAttribute`